src/components/calcite-flow/calcite-flow.stories.ts

```diff
@@ -66,6 +66,10 @@ const createFlowItemAttributes: (group: string) => Attributes = (group) => {
     {
       name: "intl-close",
       value: text("intlClose", TEXT.close, group)
+    },
+    {
+      name: "style",
```

Comment:
Is this intentional? If it's needed for the story to render properly, let's move it to the story creation logic.

Applies to other component stories.